lib/launcher.ts

```diff
@@ -296,9 +300,13 @@ let initFn = function(configFile: string, additionalConfig: Config) {
               }
               return cleanUpAndExit(exitCode);
             })
-            .done();
+            .catch(err => setTimeout(() => {
```

Comment:
This is a tough one because there's no equivalent to `done` on the native side. I think we should either move these into a helper function or talk about returning this deferred so the exception can bubble back up the chain.